src/util.ts

```diff
@@ -24,3 +26,23 @@ export const getProjectIdFromSentryDsn = (dsn: string) => {
   const search = splitUrlIntoParts(dsn)[5];
   return search.replace('/', '');
 };
+
+const isError = (exception: string | Error): exception is Error => {
+  return (exception as Error).message !== undefined;
+};
+
+/**
+ * Get the message and name properties from the original exception
+ * @param {EventHint} hint
+ */
+export const getOriginalExceptionProperties = (hint: EventHint) => {
+  let name = '';
+  let message = '';
```

Comment:
@patrick-fs is it necessary to return an empty string for these variables when we don't set them? I think it would look better and be cleaner if we just omitted them by returning an empty object when we can't get the fields.

Reply:
this is done